src/7-to-8/major-changes/compatibility-mode.rst

```diff
@@ -1,27 +1,56 @@
-.. _AutoConfigUpgrades:
+.. _728.compatibility_mode:
 
-Cylc Configuration Upgrader
-===========================
+Cylc 7 Compatibility Mode
+=========================
 
-.. admonition:: Does This Affect Me?
+.. admonition:: Does This Change Affect Me?
    :class: tip
 
-   This may affect you if you are working with workflows written before Cylc 8,
-   and you see warnings or an ``IllegalItemError`` when validating.
+   This will affect you if you want to run run Cylc 7 workflows (using the
+   ``suite.rc`` filename) using Cylc 8.0
 
 Overview
 --------
 
-Cylc has a built in configuration upgrader. **Cylc 8** can upgrade Cylc 7
-workflows on the fly if they use up-to-date Cylc 7 syntax, but *not
-if validation with Cylc 7 gives deprecation warnings*.
+The old ``suite.rc`` filename triggers a backward compatibility mode in which:
 
-Solution
---------
+- :term:`implicit tasks <implicit task>` are allowed by default
+
+  - (unless a ``rose-suite.conf`` file is found in the :term:`run directory`
+    for consistency with ``rose suite-run`` behaviour)
+  - (by default, Cylc 8 does not allow implicit tasks)
+
+- :term:`cycle point time zone` defaults to the local time zone
+
+  - (by default, Cylc 8 defaults to UTC)
+
+- waiting tasks are pre-spawned to mimic the Cylc 7 scheduling algorithm and
+  stall behaviour, and these require
+  :term:`suicide triggers <suicide trigger>`
+  for alternate path :term:`branching <graph branching>`
```

Comment:
(this code was moved rather than written for this PR)

I think that should be "graph branching" to push home to documented term:

```suggestion
for alternate :term:`graph branching`
```

Reply:
👍🏼